src/main/java/ladder/domain/RewardsOfPlayers.java

```diff
@@ -0,0 +1,41 @@
+package ladder.domain;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.IntStream;
+
+public class RewardsOfPlayers {
+    private final Map<String, String> rewardsOfPlayers;
+
+    public RewardsOfPlayers(List<Player> resultPlayers, Results results) {
+        rewardsOfPlayers = new HashMap<>();
+        IntStream.range(0, resultPlayers.size())
+                .forEach(location ->
+                        rewardsOfPlayers.put(
+                                getNameByLocation(resultPlayers, location),
+                                results.getResultReward(location)
+                        )
+                );
```

Comment:
`Stream`이 아직 생소하신것 같아 팁을 드리면 아래와 같이 한번에 생성하는 방법도 있습니다.

```suggestion
        rewardsOfPlayers = resultPlayers.stream()
                .collect(Collectors.toMap(
                        player -> getNameByLocation(resultPlayers, player.location()),
                        player -> results.getResultReward(player.location()))
                );
```